pkg/madmin/obd.go

```diff
@@ -221,11 +222,13 @@ var OBDDataTypesList = []OBDDataType{
 
 // ServerOBDInfo - Connect to a minio server and call OBD Info Management API
 // to fetch server's information represented by OBDInfo structure
-func (adm *AdminClient) ServerOBDInfo(ctx context.Context, obdDataTypes []OBDDataType) <-chan OBDInfo {
+func (adm *AdminClient) ServerOBDInfo(ctx context.Context, obdDataTypes []OBDDataType, deadlineSeconds time.Duration) <-chan OBDInfo {
 	respChan := make(chan OBDInfo)
 	go func() {
 		v := url.Values{}
 
+		v.Set("deadlineseconds", fmt.Sprintf("%v", deadlineSeconds.Truncate(1*time.Second).Seconds()))
```

Comment:
```suggestion
		v.Set("deadlineseconds", deadlineSeconds.Truncate(1*time.Second).String())
```

Reply:
the `.String()` uses a human readable format, like `30m20s`.

Reply:
and I'm actually printing the output of `.Seconds()`, which is float64

Reply:
yeah, you can directly use. 
```
v.Set("deadlineseconds", deadlineSeconds.Truncate(1*time.Second).String())
```

Reply:
done